yaml_edit/lib/src/editor.dart

```diff
@@ -0,0 +1,632 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
```

Comment:
License header: https://github.com/google/dart-neats/blob/73ada3d69f092cef445dd54e4e79953f8af3b969/safe_url_check/example/main.dart#L1-L13

Reply:
Also feel free to update the date to 2020, and add your name as author... given that you're the original author :D